lib/_http_client.js

```diff
@@ -297,6 +311,21 @@ function ClientRequest(input, options, cb) {
 Object.setPrototypeOf(ClientRequest.prototype, OutgoingMessage.prototype);
 Object.setPrototypeOf(ClientRequest, OutgoingMessage);
 
+Object.defineProperty(ClientRequest.prototype, 'aborted', {
+  get() {
+    return this[kWritableState].destroyed;
+  },
+  set(val) {
+    this[kWritableState].destroyed = val;
```

Comment:
This is for backwards compat. Setting this can cause strange behaviour. Can we remove or deprecate it?